src/nvim/window.c

```diff
@@ -5543,9 +5543,10 @@ static void frame_setheight(frame_T *curfrp, int height)
   if (curfrp->fr_parent == NULL) {
     // topframe: can only change the command line
     if (height > ROWS_AVAIL) {
-      // If height is greater than the available space, try to create space for the frame by
-      // reducing 'cmdheight' if possible, while making sure `cmdheight` doesn't go below 1.
-      height = MIN(ROWS_AVAIL + (p_ch - 1), height);
+      // If height is greater than the available space, try to create space for
+      // the frame by reducing 'cmdheight' if possible, while making sure
+      // `cmdheight` doesn't go below 1.
```

Comment:
Last year we changed the style guide to allow lines up to 100 chars long. Don't need to reformat this comment.

Reply:
Oh, I have not know this.
I just found the comment line is too long.